Makefile

```diff
@@ -54,6 +61,12 @@ build-image-and-push-linux-amd64: init-buildx
 		--platform linux/amd64 \
 		-t $(REGISTRY)/$(IMG):$(TAG)_linux_amd64 --push . --target debian-base; \
 	}
+	{                                                                   \
+	set -e ;                                                            \
+	docker buildx build \
+		--platform linux/amd64 \
+		-t $(REGISTRY)/$(IMG_NODE_UPDATE_CONTROLLER):$(TAG)_linux_amd64 --push . --target debian-base-node-update-controller; \
+	}
```

Comment:
No need to build node-update-controller every time when csi driver is built.
Create separate target for building node-update-controller image.